eq-author/src/App/QuestionnaireDesignPage/MainNavigation/index.js

```diff
@@ -234,7 +232,4 @@ UnwrappedMainNavigation.propTypes = {
   }).isRequired,
 };
 
-export default flowRight(
-  withQuestionnaire,
-  withRouter
-)(UnwrappedMainNavigation);
+export default withRouter(UnwrappedMainNavigation);
```

Comment:
Now this is a functional comp. could use `useRouter` or `useMatch` or whatever instead of using the HOC (if you want)